code/AdobeEdge.brs

```diff
@@ -288,10 +291,13 @@ function AdobeSDKInit() as object
                 end function,
                 ' dispatch events to the task node
                 dispatchEvent: function(event as object) as void
-                    _adb_log_debug("dispatchEvent() - " + FormatJson(event))
-                    if m.taskNode <> invalid then
-                        m.taskNode[m.cons.TASK.REQUEST_EVENT] = event
+                    _adb_log_debug("dispatchEvent() - Dispatching event:(" + FormatJson(event) + ")")
+                    if m.taskNode = invalid then
+                        _adb_log_debug("dispatchEvent() - Cannot Dispatch event with uuid:(" + FormatJson(event.uuid) + "), task node instance is invalid.")
```

Comment:
This should only happen after calling `shutdown` API and still holding the SDK instance somewhere, can we reword it like" do not call APIs after the SDK is shut down."?